### PR TITLE
added search

### PR DESCRIPTION
**Pull request by danielptm**

### CODE REVIEW

File: app/src/main/java/com/gatech/asl_dictionary/Search.kt [hunk 1]
```diff
@@ -55,6 +75,28 @@ fun Search(navigationToSecondScreen: (String) -> Unit) {
                 }
             }
         }
+
+
+        var inputText by remember { mutableStateOf("") }
```
Comment (Tim-Tscheppe) [inline app/src/main/java/com/gatech/asl_dictionary/Search.kt:80]: TODO: check if inputText is in Signs.json and do whatever clickButton(signEntry) does

File: app/src/main/java/com/gatech/asl_dictionary/Search.kt [hunk 2]
```diff
@@ -55,6 +75,28 @@ fun Search(navigationToSecondScreen: (String) -> Unit) {
...
+        Spacer(modifier = Modifier.height(420.dp))
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .align(Alignment.CenterHorizontally)
+        ){
+            TextField(
+                value = inputText,
+                onValueChange = { inputText = it },
+                placeholder = { Text("Search") }
```
Comment (Tim-Tscheppe) [inline app/src/main/java/com/gatech/asl_dictionary/Search.kt:91]: TODO: Make this button not touch the text input field.

**Review by Tim-Tscheppe — APPROVED** [review]
LGTM

File: app/src/main/java/com/gatech/asl_dictionary/Search.kt (hunk 1 above)
Comment (danielptm) [reply app/src/main/java/com/gatech/asl_dictionary/Search.kt:80]: Yeah. I was going to implement this separately. Actually im working on it now.

Comment (danielptm) [reply app/src/main/java/com/gatech/asl_dictionary/Search.kt:80]: Yeah. I was going to implement this separately. Actually im working on it now.

**Pull request merged**